public/index.js

```diff
@@ -8,3 +8,10 @@ function makeRequest(url, callback) {
   xhr.open('GET', url);
   xhr.send();
 }
+
+// event listener
+var input = document.getElementById('input-box');
+input.addEventListener('keyup', function(event) {
+  var url = '/?word=' + input.value;
```

Comment:
maybe we should send the query to a different endpoint? e.g. '/search/q=' ? let's talk